code/src/Components/FourthQuestion.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import Button from './Button.js'
+
+const FourthQuestion = ({ userExpectations, onExpectationsChange, section, setSection, printString, setPrintString }) => {
```

Comment:
I tried to understand why this one isn't working but the fifth question with radiobuttons is working..
The difference that I found is `printString` and `setPrintString` - these are not included in the fifth question. Try to remove them to see if that works :)

Reply:
They work now, not really sure what I did. 
It was not the `printString` though. That was an effort by me to try to print the value out in normal text in the summary, but I couldn't get that to work either 🙈. Removed them for now anyhow.